### DISCUSSION TITLE
专业版手机上突然不能用了，电脑端能用

### DISCUSSION THREAD

**Discussion by jasonatcn** [getlantern/lantern-forum-cn#216] [Q&A]
用华为手机，最近鸿蒙系统升级后就不能用了，之前稳定用了有差不多一年了。尝试重新安装最新版本，使用专业版依然无法使用（会显示链接上，但是就是不能用）。
再尝试卸载掉清除全部数据后安装，发现安装后第一次打开用，免费版本反而可以连接到服务器并可以正常使用，再切换专业版又无法使用了（同样是显示链接上但不能用，并且有时候显示连接上但服务器位置没有任何显示），什么情况？？

**Comment by MaxMKMax** [comment]
@jasonatcn 您好，谢谢您的反馈。最近鸿蒙出现一些问题, 蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
